### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by MikoZhu**
I have added and deleted some pictures in the assets folder.
npm install react-slideshow-image
add an english.json file in the translation part
reusable components are all in the "components folder"

### CODE REVIEW

File: frontend/src/pages/home/Stories.jsx
Comment (VAstrom) [inline frontend/src/pages/home/Stories.jsx:1]: The images in the slideShow don't show for me.

File: frontend/src/pages/home/NiceToMeet.jsx
Comment (VAstrom) [inline frontend/src/pages/home/NiceToMeet.jsx:1]: The image doesn't show for me. I guess we're going to have our own pictures here?

File: frontend/src/pages/home/home.module.css [hunk 1]
```diff
@@ -1,29 +1,100 @@
...
+    border-radius:30px;
+    padding:0 1rem;
+    width:160px;
 }
 
-.linkContainer{
-    margin-right: 1rem;
-    text-decoration: none;
-    color:var(--main-color)
+.half{
```
Comment (VAstrom) [inline frontend/src/pages/home/home.module.css:44]: In the Figma file this image has its center between the text box to the left and the image to the right, and the bottom is aligned with the thin line on the page. On my screen the half circle isn't aligned in this way. The bottom is slightly below the line and the half circle is positioned more to the right.

**Review by VAstrom — COMMENTED** [review]
I didn't notice any obvious bugs when trying the desktop version apart from some of the images not showing.

The mobile page and the tablet version: 
When clicking on the hamburger menu the other links appear but the hamburger menu is still visible. 
How far is the green color supposed to stretch behind the product images for these versions?

**Comment by MikoZhu** [comment]
fixed the circle part and profileController and userController's typo error. (did the same as Veronica's)

File: frontend/src/pages/home/home.module.css (hunk 1 above)
Comment (MikoZhu) [reply frontend/src/pages/home/home.module.css:44]: have solved the problem

**Pull request merged**